src/config/entities/__tests__/configuration.ts

```diff
@@ -204,6 +204,8 @@ export default (): ReturnType<typeof configuration> => ({
     },
   },
   redis: {
+    user: process.env.REDIS_USER || undefined,
+    pass: process.env.REDIS_PASS || undefined,
```

Comment:
These are `undefined` by default.

```suggestion
    user: process.env.REDIS_USER,
    pass: process.env.REDIS_PASS,
```

Reply:
Done
https://github.com/safe-global/safe-client-gateway/pull/2222/commits/9e76bd1f48294ecc71a0e57f6d30cf1a18d746c0